pkg/sharer/sharer.go

```diff
@@ -238,13 +266,15 @@ func NewSQLSharer(logger log.Logger, driver string, dsn string, createTables boo
 }
 
 func (s SQLSharer) cleanupOldLinks(retention time.Duration) (int64, error) {
-	timestampFn := "DATETIME"
-	if s.driver == "mysql" {
-		timestampFn = "TIMESTAMP"
+	var query string
+	if s.driver == "postgres" {
```

Comment:
With three variants, a `switch` starts to feel more natural:

```go
switch s.driver {
case "postgres":
  // ...
case "mysql":
  // ...
default:
  // ...
}
```

Reply:
Right, thank you for pointing. Fixed